src/actors/containerization/services/wordpress/wordpress_generator/wordpress_generator.py

```diff
@@ -0,0 +1,57 @@
+import json
+import sys
+import os
+from subprocess import check_output, CalledProcessError
+from jinja2 import Template
```

Comment:
```jinja2``` is not module from standard library, we were talking about this Makefile to install dependencies, although I don't remember what we decided... let's talk about this, I remember @shaded-enmity was resolving similar issue.

Reply:
Yeah, the problem here is that Jinja2 has its own dependencies (although only 2), nevertheless you'd need to bundle all them.

Since Jinja2 is a dependency of Ansible, it'll always be accessible locally, so I'd probably flesh out this part of code and chuck it into an actor that is executed locally.